### DISCUSSION TITLE
h-screen not working

### DISCUSSION THREAD

**Discussion by ta-vroom** [tailwindlabs/tailwindcss#7881] [Help] [answered]
**What version of Tailwind CSS are you using?**
v3.0.23

**What build tool (or framework if it abstracts the build tool) are you using?**
tailwind-cli

**What version of Node.js are you using?**
v17.6.0

**What browser are you using?**
Firefox

**What operating system are you using?**
WSL 

**Describe your issue** 

I have a line in my project like this (React): 
`<div className='flex flex-row-reverse items-center min-h-screen'>`

I would expect for min-h-screen to be added to the resulting css file, but this does not seem to happen. I can use the minified css from the cdn and it works just fine, but I would rather not use that. 

I suspect this might have to do with my tailwind config, so I will link that. I am not sure if this is related to #7439.

```
module.exports = {
  content: ["./{src,public}/**/*.{html,js,ts,jsx,tsx}", "./src/Pages/*.tsx", "./src/Components/*.tsx", {raw: '<div class=min-h-screen', extension: 'html' }],
  theme: {
  extend: {},
  },
  plugins: [
  ]
}
```

**Comment by ta-vroom** [comment, marked as answer]
For anyone else having this problem or something similar: it turned out that I copied the wrong command from my command history. I was importing `/src/css/styles.css` into my project, but tailwind was generating the css at `css/styles.css`

**Reply by zhugexinxin** [reply]
me too
